components/hab/src/command/cli/setup.rs

```diff
@@ -311,3 +351,60 @@ fn opt_out_analytics(ui: &mut UI, analytics_path: &Path) -> Result<()> {
     ui.br()?;
     result
 }
+
+#[cfg(windows)]
+fn binlink_is_on_path(binlink_path: &Path) -> bool {
+    match RegKey::predef(HKEY_LOCAL_MACHINE).open_subkey_with_flags(
+        "System\\CurrentControlSet\\Control\\Session Manager\\Environment",
+        KEY_READ,
+    ) {
+        Ok(env) => {
+            let path: String = env
+                .get_value("path")
+                .expect("could not find a machine PATH");
+            env::split_paths(&path).any(|p| p == binlink_path)
+        }
+        _ => false,
+    }
+}
+
+/// this sets the permanent machine PATH and not
+/// the path of this process. These are maintained
+/// in the Windows registry
+#[cfg(windows)]
+fn set_binlink_path(binlink_path: &Path) -> Result<()> {
+    if !binlink_is_on_path(binlink_path) {
+        let hklm = RegKey::predef(HKEY_LOCAL_MACHINE);
+        let env = hklm.open_subkey_with_flags(
+            r"System\CurrentControlSet\Control\Session Manager\Environment",
```

Comment:
I didn't mean to say with 
https://github.com/habitat-sh/habitat/pull/5513#discussion_r213380316 that you had to change this, I just thought it was useful to know for a windows context.

However, whichever way you go, I would like it to be consistent with the  path on L358.